frontend/src/components/Metrics/ConfigStep/PipelineTool/style.tsx

```diff
@@ -0,0 +1,60 @@
+import { styled } from '@mui/material/styles'
+import { Backdrop, Button, FormControl, TextField } from '@mui/material'
+import { theme } from '@src/theme'
+
+export const PipelineToolSection = styled('div')({
```

Comment:
```suggestion
export const PipelineToolSection = styled.div({
```

Reply:
all html tags are recommended used by this way in styled component. Same as h2, form below.

Reply:
I think that it will be fixed in all 'style.tsx' files later about styled component.